homeassistant/components/sense/const.py

```diff
@@ -12,6 +12,7 @@
 SENSE_DEVICE_UPDATE = "sense_devices_update"
 SENSE_DEVICES_DATA = "sense_devices_data"
 SENSE_DISCOVERED_DEVICES_DATA = "sense_discovered_devices"
+SENSE_TRENDS_COORDINATOR = "sense_trends_coorindator"
```

Comment:
coordinator

Reply:
Good eye!

https://github.com/home-assistant/core/pull/34181

Reply:
Glad I made it a constant instead of a magic string.